spec/models/onboarding_message_spec.rb

```diff
@@ -0,0 +1,35 @@
+require "rails_helper"
+
+describe OnboardingMessage do
+  describe "Associations" do
+    it { should have_many(:sent_messages).dependent(:destroy) }
+  end
+
+  describe "Validations" do
+     it { should validate_presence_of(:body) }
+     it { should validate_presence_of(:days_after_start) }
+     it { should validate_presence_of(:tag_list) }
+     it { should validate_presence_of(:time_of_day) }
+     it { should validate_presence_of(:title) }
+  end
+
+  describe '.date_time_ordering' do
+    it 'should display the messages in chronological order' do
+      m1 = create(:onboarding_message, days_after_start: 1, time_of_day: '2000-01-01 11:00:00 UTC')
```

Comment:
can we create these out of order so we know that the ordering is working?  I think the would return in this order by default anyway so that would increase the quality of test